pages/api/graphql-rest.page.ts

```diff
@@ -148,6 +152,13 @@ class MpdxRestApi extends RESTDataSource {
     );
     return mapExpectedMonthlyTotalReport(data);
   }
+
+  async getDesignationAccounts(accountListId: string) {
+    const { data }: { data: DesignationAccountsResponse[] } = await this.get(
+      `account_lists/${accountListId}/designation_accounts?include=organization&per_page=10000`,
```

Comment:
It doesn't look like you're using the organization anywhere. Will that be needed?
```suggestion
      `account_lists/${accountListId}/designation_accounts?per_page=10000`,
```